### DISCUSSION TITLE
Add option to scale lidar points with zoom level.

### DISCUSSION THREAD

**Discussion by drldcsta** [foxglove/community#317] [Q&A]
I think it would be better if you could zoom in on lidar data and still be able to see the points. I understand that there’s a lo

Lidar data looks great here:
<img width="625" alt="Screen Shot 2022-03-31 at 5 40 50 PM" src="https://user-images.githubusercontent.com/4950919/161153901-2dc1c1e0-95d7-41e8-8b6a-5f86513f317a.png">
But not so great here (Zoomed in image so you can sort of see the points)
<img width="629" alt="Screen Shot 2022-03-31 at 5 41 21 PM" src="https://user-images.githubusercontent.com/4950919/161153990-206d59df-04cb-4774-b07e-f4b8dc5c2736.png">
<img width="623" alt="Screen Shot 2022-03-31 at 5 42 02 PM" src="https://user-images.githubusercontent.com/4950919/161154092-460da80c-cdf4-4435-877e-cba765e9844d.png">

**Comment by jtbandes** [comment]
Seems like we would want this to be a setting, it feels unlikely that every user would want their points to zoom with the camera.

**Comment by amacneil** [comment]
Could we do something intelligent so that only after a certain zoom level / density the points start getting larger?

**Comment by AndrewNolte** [comment]
I think this can be done as a webGL shader. I did something similar or a graphics project where I wanted the viz to look good even when zoomed in. And I think it'll end up looking more natural when zooming in, as the physical size of the points wont change anymore. Right now they are changing, as the point size is defined in relation to the screen, not the 3d environment
